types/express-http-proxy/index.d.ts

```diff
@@ -11,6 +11,12 @@ import { RequestOptions, IncomingHttpHeaders, OutgoingHttpHeaders } from "http";
 import { Response } from "express-serve-static-core";
 
 interface ProxyOptions {
+    /**
+     * The byte limit of the body. This is the number of bytes or any string
+     * format supported by `bytes`, for example `1000`, `'500kb'` or `'3mb'`.
+     * See https://github.com/stream-utils/raw-body/blob/master/index.d.ts
+     */
+    limit?: number | string | null;
```

Comment:
As limit is optional already, why `| null`?

Reply:
If `null` is supported please add a test case, if not please  just remove.

Reply:
@ulrichb I added it since it was part of the type definition from the `raw-body` package. But yes, since limit is optional it makes sense to leave it out for now.
PR has been updated.